collections-csharp/MergingDictionariesInCSharp/MergingDictionariesInCSharp/Methods/UnionMethod.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace MergingDictionariesInCSharp
+{
+    public partial class Methods
+    {
+        public static Dictionary<TKey, TValue> UnionMethod<TKey, TValue>(params Dictionary<TKey, TValue>[] dictionaries)
```

Comment:
We should add a test for the expected ArgumentException when the dictionaries key sets are not mutually exclusive.

Reply:
Added Argument Exception test